soc/renesas/ra/ra2/Kconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024 MUNIC SA
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RA2
```

Comment:
We don't recommend using "ra2" as SoC series because there is risky for future MCUs. We considered for development of future RA MCU and recommend using RA2L1 as SoC Series:
SoC Family: RA
SoC Series: RA8M1, RA8D1, RA8T1, RA6M1, .... RA2L1
SoC: R7FA2L1AB3CFP, R7FA2L1AB3CFP, ...
Board: EK-RA8M1, EK-RA2L1, ....